### PR TITLE
Node version

### PR DESCRIPTION
**Pull request by domtra**
had to upgrade dependencies in order to get node v8 to work. skipped upgrading all standard related packages due to a new linting rule that conflicts with our usage of webpack. see https://github.com/standard/standard/issues/937

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,31 +12,31 @@
...
+    "file-loader": "^0.11.2",
     "glob": "^7.1.1",
     "gulp": "^3.9.1",
-    "gulp-changed": "^2.0.0",
+    "gulp-changed": "^3.1.0",
     "gulp-changed-in-place": "^2.0.3",
     "gulp-if": "^2.0.2",
     "gulp-notify": "^3.0.0",
-    "gulp-phpcs": "^1.3.0",
+    "gulp-phpcs": "^2.0.0",
```
Comment (Qakulukiam) [inline package.json]: Nice, looks like we can get rid of this now: https://github.com/flyntwp/flynt-starter-theme/pull/192

File: package.json [hunk 2]
```diff
@@ -61,7 +61,7 @@
     "stylint-stylish": "^1.4.0",
     "stylus": "^0.54.5",
     "uglify-loader": "^2.0.0",
-    "webpack": "^2.2.1"
+    "webpack": "^3.0.0"
```
Comment (Qakulukiam) [inline package.json]: wow, no adjustments necessary at all? 😮

Comment (domtra) [inline package.json]: nope

File: package.json (hunk 1 above)
Comment (domtra) [inline package.json]: looks like it

**Pull request merged**